### ISSUE TITLE
Uncaught (in promise) TypeError: _.contains is not a function

### ISSUE THREAD

**Issue by dejanbatanjac** [tensorflow/tensorboard#1684]
TensorBoard running under Ubuntu Chromium Web Browser.
Switch to Show Code mode, select a python file and the error like this one occurs:
```
(index):12064 Uncaught (in promise) TypeError: _.contains is not a function
    at (index):12064
    at Array.filter (<anonymous>)
    at HTMLElement._filterFileTracebacksByDebugWatches ((index):12064)
    at (index):12058
```

**Comment by stephanwlee** [comment]
Hi @dejanbatanjac can you please fill out the issue template? Which version of TensorBoard are you using?
Thanks!

**Comment by dejanbatanjac** [comment]
TensorBoard VERSION = '1.12.0'
Python 2.7.15rc1
@stephanwlee can you link to the issue template? I would be glad to fill it out.

**Comment by stephanwlee** [comment]
Issue template looks something like this when you create one:
![image](https://user-images.githubusercontent.com/2547313/49817566-6ed6fa80-fd25-11e8-86e8-a36da6442300.png)

It looks like the issue is fixed in #1550. The fix should be part of next release. Thanks!

**Comment by dejanbatanjac** [comment]
@stephanwlee You may check out the previous post. I updated the issue.

**Comment by wchargin** [comment]
@dejanbatanjac: Thanks. The stack trace that you provided refers to the
function that was patched in #1550, the PR that @stephanwlee linked
earlier. There are currently no references to `_.contains` in our code.

As Stephan said, this fix will be included in the next release.
